code/ui/blocks/src/components/Preview.tsx

```diff
@@ -198,6 +187,7 @@ export const Preview: FC<PreviewProps> = ({
   isExpanded = false,
   additionalActions,
   className,
+  layout = 'padded',
```

Comment:
I think you need to update the preview stories to have this prop, eg:

https://github.com/storybookjs/storybook/blob/2cceaadef83bcc109af4144030a5e79af1a9d48f/code/ui/blocks/src/components/Preview.stories.tsx#L156-L172